sdk/digitaltwins/Azure.DigitalTwins.Core/src/RequestOptions.cs

```diff
@@ -17,6 +17,11 @@ public class RequestOptions
         /// only if this ETag matches the value maintained by the server, indicating that the entity has not been modified since it was last retrieved.
         /// To force the operation to execute only if the entity exists, set the ETag to the wildcard character '*'. To force the operation to execute unconditionally, leave this value null.
         /// </summary>
+        /// <remarks>
+        ///   <para>
```

Comment:
I think you don't need to use <para> unless you are trying to split up multiple paragraphs

Reply:
I see. Ok i'll remove it. And we can fix it based on how it is rendered.